lib/pure/json.nim

```diff
@@ -1087,6 +1129,37 @@ when isMainModule:
   except:
     assert(false, "EInvalidIndex thrown for valid index")
 
+  # Generator:
+  var j = %* [{"name": "John", "age": 30}, {"name": "Susan", "age": 31}]
+  assert j == %[%{"name": %"John", "age": %30}, %{"name": %"Susan", "age": %31}]
+
```

Comment:
Speaking of which, can we not distinguish between '%*' and '%'? That makes it much easier to use IMHO. Operator overloading should be able to handle that, right?